app/models/adoption_application.rb

```diff
@@ -0,0 +1,18 @@
+class AdoptionApplication < ActiveRecord::Base
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  VALID_PHONE_REGEX = /[0-9]{10}/
+  before_save :downcase_email
+  validates :name, presence: true
+  validates :address, presence: true
+  validates :city, presence: true
+  validates :cell, presence: true,
+                   format: { with: VALID_PHONE_REGEX , message: "Please enter phone number without spaces or hyphens."}
```

Comment:
Line is too long. [119/80]<br>Space found before comma.<br>Space inside } missing.